### PR TITLE
Country selection error tweaks

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Task

Based on user testing feedback we want to remove help message and tweak the copy of the no results error message.

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [ ] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [x] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [x] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [x] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1181-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/CountrySelector/index.js [hunk 1]
```diff
@@ -96,6 +96,32 @@ class CountrySelection extends Component<Props, State> {
     return enabledIdentityDocuments.length === 1 && documentType !== 'passport'
   }
 
+  getNoResultsTextForDropdown = () =>
+    parseTags(
+      this.props.translate(`country_selection.inline_error`),
```
Comment (kopijunkie) [inline src/components/CountrySelector/index.js:101]: Maybe `country_selection.dropdown_error` would be a better key name for this.

**Review by kopijunkie — APPROVED** [review]

**Review by ethan605 — APPROVED** [review]
LGTM!

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1181-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**